shedskin/cpp.py

```diff
@@ -599,45 +612,48 @@ def do_comments(self, child):
             for n in self.gx.comments[child]:
                 self.do_comment(n)
 
-    def visitContinue(self, node, func=None):
+    def visit_Continue(self, node, func=None):
         self.output('continue;')
 
-    def visitWith(self, node, func=None):
+    def visit_With(self, node, func=None):
```

Comment:
Could you please write all lowercase when using `_`, in this case `visit_with`?
This matches the naming conventions in the other modules.